NEWS.md

```diff
@@ -2,6 +2,11 @@
 
 ## 0.8.29 (in development)
 
+* Introduced support for publishing to Posit Cloud. This feature is currently
+  in closed beta and requires access to an enabled account on Posit Cloud.
+  See [Posit Cloud's Announcement](https://posit.cloud/learn/whats-new#publishing)
```

Comment:
I don't see this anchor on the page; is it still pending?

Reply:
Yup. @robbyshaver can comment on ETA, but my guess is soon enough that we don't really need to have a hard dependency.

Reply:
PR is waiting for approval  ...